### PR TITLE
Make storybook build stop complaining unnecessarily

### PR DESCRIPTION
**Pull request by pcardune**
Depending on how dashboard assets get compiled, the resulting files might end up
in different places. This was causing the storybook build script to log an error
message even though it did not fail. We were already correctly ignoring the
return code for the cp command, but the error message was left intact. Now we
send the error message, which we likewise do not care about, to /dev/null

### CODE REVIEW

File: apps/script/deploy-storybook.sh [hunk 1]
```diff
@@ -19,8 +19,8 @@ rm -rf $DIR_TO_DEPLOY
 echo "Compiling application.css using rails asset pipeline"
 pushd ../dashboard
 rails assets:precompile
-cp public/assets/css/application-*.css ../apps/build/package/application.css || :
-cp public/assets/application-*.css ../apps/build/package/application.css || :
+cp public/assets/css/application-*.css ../apps/build/package/application.css 2>/dev/null || :
+cp public/assets/application-*.css ../apps/build/package/application.css 2>/dev/null || :
```
Comment (islemaster) [inline apps/script/deploy-storybook.sh]: Wait, I don't understand... if these are going to fail, why don't we just remove them?  Or are we expecting at least one of these to succeed - and if so, can we check for that?  I thought they both failed in the log I sent you (but can't check right now due to the S3 outage).

Comment (pcardune) [reply apps/script/deploy-storybook.sh]: only one of them should fail. I would check that explicitly, but bash makes my head hurt. If someone around here can show me how to do it in bash, I will change it.

Comment (islemaster) [reply apps/script/deploy-storybook.sh]: Would they ever both succeed?  If not, could do `A || B` and not swallow the result.

**Comment by pcardune** [comment]
ok, updated. PTAL?

**Pull request merged**